### DISCUSSION TITLE
Android WiFi scanning throttling

### DISCUSSION THREAD

**Discussion by pippo73** [VREMSoftwareDevelopment/WiFiAnalyzer#471] [Q&A] [answered]
Hello all,
When I run this app I get a warning that I have some kind of wifi restriction. 

Could you explain me what is it? 🤔 

![markup_1000052304.png](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/assets/757121/f8c6a48b-3ed4-4cf2-ba39-02744d65f3f6)

**Comment by VREMSoftwareDevelopment** [comment, marked as answer]
[FAQ](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer#faq)
[Android Wi-Fi scanning throttling](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/wiki/Android-WiFi-scanning-throttling)
